### ISSUE TITLE
krp|squarespace plan purchase

### ISSUE THREAD

**Issue by swcalise** [fillungo/client-KRP#8]
- [x] determine cost
- [x] communicate cost to KRP
- [x] purchase plan

**Comment by swcalise** [comment]
@Callus4815 can you give very specific costs for this so I can communicate a final $$ to them? 
Then in the setup/purchase, if they provide a credit card, can you do everything or will there be things they need to do?

**Comment by Callus4815** [comment]
Squarespace -- 172.80/yr for business plan(I can do)

**Comment by Callus4815** [comment]
Hosting and Domain registrar are variables at the moment. Squarespace estimates $20-$70 to switch a domain registrar.

**Comment by swcalise** [comment]
grazie

**Comment by swcalise** [comment]
follow up: how will emails work - they have @kavichreynolds.com and wish to maintain

**Comment by Callus4815** [comment]
They can keep it at their current hosting with yahoo. The squarespace that comes with the plan we will purchase offers Gsuite free for a year. We should turn off hosting at yahoo and keep the email service, to not interrupt service(if thats available). It should be. Currently they are spending $35 a quarter for yahoo. Removing the hosting part may lower that per quarter to counter the new expense of squarespace.

**Comment by swcalise** [comment]
How can we confirm the turn off hosting, but keep email option?
Should I ask for their Yahoo login credentials?
Are you in correspondence with Bret already? maybe he'd be able to provide. FYI Bret will not be as available starting July 1, so hit him up with anything sooner than later.

**Comment by Callus4815** [comment]
I got them already from Brett yesterday. I just havent had time to sign in
and get the problem rolling.

**Comment by Callus4815** [comment]
I got yahoo login credentials and the domain registrar credentials.

**Comment by Callus4815** [comment]
Im going to have to call Yahoo small business to talk about cancelling hosting but keeping the email(if thats even a service they offer). They are available 6am-5pm for phone calls(no chat available). This may be tough for me, but if one of you have time. Basically we just need to ask if it can be done. And some direction on how. The number is 1-866-781-9246.

**Comment by swcalise** [comment]
ok cool, I can handle to save your hours.
but are we even sure they 2 are connected: their @kavichreynolds.com email
and Yahoo?

**Comment by Callus4815** [comment]
I think so. Within Yahoo there is a link to "Email" and it has all the
email accounts. Unless Im mistaken I think that means they connected, but
that was something I was going to ask them.

**Comment by swcalise** [comment]
I called Yahoo and got some good info.
Currently they have a Hosting plan. 
To move off that:
1. Backup site first - using filezilla etc
2. downgrade to business mail plan - to maintain email less than $2/month
[https://smallbusiness.yahoo.com/mail](url)
3. get a and c-name records from squarespace then input into Yahoo domain mng of the portal: here's an article on how [https://help.smallbusiness.yahoo.net/s/article/SLN17916](url)
4. then I look to you on how to coordinate the actual switch-over when the time comes bc they were wanting to consolidate things to squarespace going fwd.

**Comment by Callus4815** [comment]
awesome. Thank you. They dont really tell you how to leave their service on
the website. Ha.

**Comment by swcalise** [comment]
I got approval from Steve to purchase and bill them.
@Callus4815 do you want a credit card to purchase this with? It will be a pass through cost to KRP

**Comment by Callus4815** [comment]
Yes please, This will be set to renew every year until we turn it off.

**Comment by swcalise** [comment]
ok here you go:
Scott W Calise
4147202287903015
exp 12/21
code: 024

**Comment by Callus4815** [comment]
@swcalise Whats the Postal code associated with this account. Squarespace wants it.

**Comment by swcalise** [comment]
think it should be their business address or tied to my credit card?
If credit card; 27517
If KRP: 90068

**Comment by Callus4815** [comment]
the one tied to the card...thanks.

**Comment by Callus4815** [comment]
Purchased, I will email the receipt i received. It is also right here https://caper-tunny-rj6j.squarespace.com/config/invoices/stripe/in_DDFAXKHhfqrno5
And since you two are admins you can see it in Settings -> Billing & Account -> Invoices
